src/server/index.js

```diff
@@ -24,6 +24,7 @@ options.loginRoute = `${approute}/login`;
 options.keysetRoute = `${approute}/keys`;
 options.invalidTokenRoute = `${approute}/invalidtoken`;
 options.sessionTimeoutRoute = `${approute}/sessiontimeout`;
+options.devMode = true;
```

Comment:
@jackvanb Can you make this so it is in devMode if the .env MODE is development?

I am assuming once we get https working we don't need devMode to be true.

Reply:
After switching to load the lti tool in a new window in CCLE, this is no longer needed to work correctly.